app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -1,38 +1,36 @@
 package edu.uc.hickmadc.customnotifications
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.ListView
+import androidx.appcompat.app.AppCompatActivity
 import androidx.navigation.Navigation
 import androidx.navigation.findNavController
 import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
-
 class MainActivity : AppCompatActivity() {
 
-    //temp data
```

Comment:
In most cases I try to avoid commenting in code like this since its easy to miss under some IDE configurations. I don't think temporary data like this justifies JavaDoc though. We'll see what the professor has to say about that, though.

Reply:
I'd go one step further an move this into a ViewModel with a stub service class.  Then, it's a given it's temp data; no need for the comment.